libnmstate/nm/dns.py

```diff
@@ -100,3 +101,12 @@ def get_config():
     if not dns_conf[DNS.SERVER] and dns_conf[DNS.SEARCH]:
         return {}
     return dns_conf
+
+
+def add_dns(setting_ip, dns_state):
+    if DNS_PRIORITY in dns_state:
```

Comment:
In PY3.7, you can do `if priority := dns_state.get(DNS_PRIORITY); ...foo = priority`.
For the simple people, we can just fetch the data once:
```
priority = dns_state.get(DNS_PRIORITY)
if priority:
    setting_ip.props.dns_priority = priority
```